packages/react-components/src/templates/ComplianceReportForm.tsx

```diff
@@ -0,0 +1,188 @@
+    },
+  });
+
+  const {
+    handleSubmit,
+    control,
+    formState: { isSubmitting },
+  } = methods;
+
+  const onSubmit = async (data: ComplianceReportFormData) => {
+    await onSave({
+      ...data,
+      manuscriptVersionId,
+    });
+
+    onSuccess();
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <main css={mainStyles}>
+        <div css={contentStyles}>
+          <Card overrideStyles={cardStyles}>
+            <Paragraph noMargin>
+              <strong>Title of Manuscript</strong>
+            </Paragraph>
+            <Paragraph>{manuscriptTitle}</Paragraph>
+            <Controller
+              name="url"
+              control={control}
+              rules={{
+                pattern: {
+                  value: new RegExp(urlExpression),
+                  message:
+                    'Please enter a valid URL, starting with http:// or https://',
+                },
+                required: 'Please enter a url.',
+              }}
+              render={({
+                field: { value, onChange },
+                fieldState: { error },
+              }) => (
+                <LabeledTextField
+                  title="URL"
+                  subtitle={'(required)'}
+                  onChange={onChange}
+                  customValidationMessage={error?.message}
+                  value={value || ''}
+                  enabled={!isSubmitting}
+                  labelIndicator={<GlobeIcon />}
+                  placeholder="https://example.com"
+                />
+              )}
+            />
+
+            <Controller
+              name="description"
+              control={control}
+              rules={{
+                required: 'Please enter a description.',
+              }}
+              render={({
+                field: { value, onChange },
+                fieldState: { error },
+              }) => (
+                <LabeledTextArea
+                  title="Compliance Report Description"
+                  subtitle="(required)"
+                  tip={
+                    <span>
+                      Add a description to the compliance report. You can format
+                      your text by using markup language.
+                    </span>
+                  }
+                  customValidationMessage={error?.message}
+                  value={value || ''}
+                  onChange={onChange}
+                  enabled={!isSubmitting}
+                />
+              )}
+            />
+          </Card>
+          <div css={buttonsOuterContainerStyles}>
+            <div css={buttonsInnerContainerStyles}>
+              <Button
+                noMargin
+                enabled={!isSubmitting}
+                onClick={() => history.goBack()}
+              >
+                Cancel
+              </Button>
+              <Button
+                primary
+                noMargin
+                submit
+                enabled={!isSubmitting}
```

Comment:
It seems the button starts as disabled when all the form is empty https://www.figma.com/design/zY4fHBAsXCiyW6gGrnS4Yy/CRN-%2F-Design-%2F-Network?node-id=24087-188526&t=KN9QcOYPgA7EeHyB-4 but maybe we could ask Padmini/João/Irene if it is supposed to be that way since it is not that consistent with other forms behavior

Reply:
Thanks, great catches! I've raised this with Padmini and she says this is the ideal behaviour. I'm updating this form (as well as the other changes raised) but there will be a subsequent ticket to update other forms.